oval_graph/client.py

```diff
@@ -1,16 +1,20 @@
 from __future__ import print_function, unicode_literals
-from datetime import datetime
+
 import re
-import oval_graph.xml_parser
-import oval_graph.oval_graph
-import oval_graph.converter
 import webbrowser
 import json
 import argparse
 import tempfile
 import os
 import shutil
 
+from datetime import datetime
```

Comment:
`datetime` is a Python standard library import, so no need to put it aside from other std imports.